java/ql/src/experimental/Security/CWE/CWE-273/UnsafeCertTrust.ql

```diff
@@ -0,0 +1,228 @@
+/**
+ * @name Unsafe implementation of trusting any certificate or missing hostname verification in SSL configuration
```

Comment:
The `@name` is a little bit long - could you perhaps shorten it slightly?

Reply:
Thanks @aschackmull . I've shortened the name attribute value.